drafts/draft-ietf-ippm-ioam-ipv6-options.xml

```diff
@@ -0,0 +1,499 @@
+      <t>In-situ Operations, Administration, and Maintenance (IOAM) records
+      operational and telemetry information in the packet while the packet
+      traverses a path between two points in the network. This document
+      outlines how IOAM data fields are encapsulated in IPv6.</t>
+    </abstract>
+  </front>
+
+  <middle>
+    <section title="Introduction">
+      <t>In-situ Operations, Administration, and Maintenance (IOAM) records
+      operational and telemetry information in the packet while the packet
+      traverses a path between two points in the network. This document
+      outlines how IOAM data fields are encapsulated in the IPv6 <xref
+      target="RFC8200"></xref>.</t>
+    </section>
+
+    <section title="Conventions">
+      <section title="Requirements Language">
+        <t>The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT",
+        "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and
+        "OPTIONAL" in this document are to be interpreted as described in BCP
+        14 <xref target="RFC2119"></xref> <xref target="RFC8174"></xref> when,
+        and only when, they appear in all capitals, as shown here.</t>
+      </section>
+
+      <section title="Abbreviations">
+        <t>Abbreviations used in this document:</t>
+
+        <t><list hangIndent="11" style="hanging">
+            <t hangText="E2E:">Edge-to-Edge</t>
+
+            <t hangText="IOAM:">In-situ Operations, Administration, and
+            Maintenance</t>
+
+            <t hangText="OAM:">Operations, Administration, and Maintenance</t>
+
+            <t hangText="POT:">Proof of Transit</t>
+          </list></t>
+      </section>
+    </section>
+
+    <section title="In-situ OAM Metadata Transport in IPv6">
+      <t>In-situ OAM in IPv6 is used to enhance diagnostics of IPv6 networks.
+      It complements other mechanisms proposed to enhance diagnostics of IPv6
+      networks, such as the IPv6 Performance and Diagnostic Metrics
+      Destination Option described in <xref target="RFC8250"></xref>.</t>
+
+      <t>IOAM data fields are encapsulated in &ldquo;option data&rdquo; fields
+      of two types of extension headers in IPv6 packets - either Hop-by-Hop
+      Options header or Destination options header. The selection of a
+      particular extension header type depends on IOAM usage, as described in
+      section 4 of <xref target="I-D.ietf-ippm-ioam-data"></xref>. Multiple
+      options with the same Option Type MAY appear in the same Hop-by-Hop
+      Options or Destination Options header, with varying content.</t>
+
+      <t>In order for IOAM to work in IPv6 networks, IOAM MUST be explicitly
+      enabled per interface on every node within the IOAM domain. Unless a
+      particular interface is explicitly enabled (i.e. explicitly configured)
+      for IOAM, a router MUST drop packets which contain extension headers
+      carrying IOAM data-fields. This is the default behavior and is
```

Comment:
We can specify it this way, but if we do shouldn't the requested Option Type "codepoint" other than "00"?
See comment below.

Reply:
Sorry, draft-ietf-ippm-ioam-ipv6-options.xml changes showed up in the pull request as my local master was not in sync with the repo. Now merged to latest. Only intended to address data draft changes in this pull request.